pkg/util/log/log.go

```diff
@@ -49,21 +50,22 @@ func SetLog(logLevel string, logfile bool, logFileName string) {
 	})
 }
 
-type MyFormatter struct {
+type myFormatter struct {
```

Comment:
I thought this was previously changed to drop the `my`

Reply:
Yes, my mistake.